### PR TITLE
Add `placeholder` to `Request`

### PR DESCRIPTION
**Pull request by tayiorbeii**
* Adds optional `placeholder` prop to `Request` to allow for an alternative `Loader` component.

### CODE REVIEW

**Comment by tayiorbeii** [comment]
@trevordmiller `yarn bump` messed up because I didn't have my upstream set, but I think I fixed it... confirm/deny?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "egghead-ui",
-  "version": "4.7.1",
+  "version": "4.7.2",
```
Comment (trevordmiller) [inline package.json:3]: I'd personally bump the major version since this is a new feature, but no worries this works :)

**Review by trevordmiller — APPROVED** [review]
Looks good. Approved. Thanks for the gif. Would be cool if we could slow down the network for the `Request` example, but this works great for now. Thanks!

**Comment by tayiorbeii** [comment]
@trevordmiller I throttled to GPRS 50kb/sec in devtools and it was still that fast!

**Pull request merged**